### ISSUE TITLE
XGBoost mixer

### ISSUE THREAD

**Issue by paxcema** [mindsdb/lightwood#1059]
If the package behavior is better LightGBM, which is segfaulting randomly, then we should create a mixer for this and replace LGBM as the default option for gradient boosting.

**Comment by paxcema** [comment]
Closed by #1066
